update_prevalence.py

```diff
@@ -1269,25 +1301,79 @@ def parse_jhu_daily_report(cache: DataCache, data: AllData, current: date) -> No
         place.cumulative_cases[current] = line.Confirmed
 
 
+# returns estimate of number of people who completed their primary
+# series (regardless of their booster status) given number of people
+# with at least one dose
+def estimate_primary_series_complete(
+    # people with at least one dose
+    partial_vaccinations: int,
+) -> int:
+    # https://covid.cdc.gov/covid-data-tracker/#vaccinations_vacc-people-additional-dose-totalpop
+    #
+    # As of 2022-09, CDC says: 68% of population has completed
+    # primary series; 49% of those who completed a primary series
+    # got first booster, 36% of those who got a first booster shot
+    # got a second.
+    #
+    # 1 dose: 263,812,108
+    # completed primary series: 224,980,931
+    # People with a First Booster Dose: 109,578,270
+    # People with a Second Booster Dose: 23,118,101
+    #
+    # Let's assume that these trends can be applied world-wide,
+    # and thus if n people had at least one dose,
+    # 224980931/263812108 = 85% of them completed the primary
+    # series.
+    primary_series_completion_ratio = 0.85
+
+    return round(partial_vaccinations * primary_series_completion_ratio)
```

Comment:
Since JHU is no longer aggregating better information from its sources, and the current microCOVID model uses (or at least reports on) primary series completion, it's up to us to come up with an estimate of it based on what they give us now.

I tested this way of estimating against an initial try where I came up with a very conservative possible estimate of vax rate (see that initial code here: https://github.com/microCOVID/microCOVID/pull/1578/files#diff-093196c543f2689ed8ed57c57bb46ce008529f882556657678049ccbc38497f9R1332-R1344)

France vax rate before: 81%
France vax rate after - conservative: 59%
France vax rate after - CDC-adjusted: 71% 
Notes: I haven't dug up ground truth to see what number is more correct here, but we're at least in the ballpark of the numbers JHU was giving out before.

US vax rate before: 68%
US vax rate after - conservative: 47%
US vax rate after - CDC-adjusted: 69%
Notes: CDC says 67.8% of US population completed primary series: https://covid.cdc.gov/covid-data-tracker/#vaccinations_vacc-people-additional-dose-totalpop - so the number before and the CDC-adjsuted numbers are as expected, given this was the training set for this super rough method.

Alabama vax rate before: 52%
Alabama vax rate after - conservative: 52%
Alabama vax rate after - CDC-adjusted: 52%
Notes: We continue to get person-level data for primary series completion for US states through JHU, not just at-least-one-dose-received, so this is as expected.